js/ui/map.js

```diff
@@ -417,6 +417,9 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
      *
      * @example
      * var features = map.queryRenderedFeatures([[10, 20], [30, 50]], { layers: ['my-layer-name'] });
+     *
+     * @example
```

Comment:
To further clarify this, we could caption the examples. Here is the (wonky) JSDoc syntax for that:

```js
/**
 * ...
 * @example <caption>Retrieve all rendered features from a single layer</caption>
 * var features = map.queryRenderedFeatures({ layers: ['my-layer-name'] });
 */